block/provider/delete_snapshot.go

```diff
@@ -46,6 +46,10 @@ func (vpcs *VPCSession) DeleteSnapshot(snapshot *provider.Snapshot) error {
 	})
 
 	if err != nil {
+		modelError, ok := err.(*models.Error)
+		if ok && len(modelError.Errors) > 0 && string(modelError.Errors[0].Code) == "snapshots_not_found" {
```

Comment:
lets create a const for all hardcoded values

Reply:
Done